boilerplate/app/components/screen/screen.tsx

```diff
@@ -44,6 +80,8 @@ function ScreenWithScrolling(props: ScreenProps) {
           style={[preset.outer, backgroundStyle]}
           contentContainerStyle={[preset.inner, style]}
           keyboardShouldPersistTaps={props.keyboardShouldPersistTaps || "handled"}
+          onContentSizeChange={onContentSizeChange}
```

Comment:
Let's guard against running the onContentSizeChange unnecessarily.

```suggestion
          onContentSizeChange={props.preset === 'auto' ? onContentSizeChange : undefined}
```